### PR TITLE
added code to generate better table, also added in exceptions to new …

### PR DESCRIPTION
**Pull request by colleend**
* added exceptions to new tests
* added code for rendering new tests
* added code to generate table ordered by Alpha, Beta, GA features
* added code to generate headers for Alpha, Beta, GA groups

### CODE REVIEW

**Review by minusworld — APPROVED** [review]
\o/

File: scripts/generate_test_matrix.py [hunk 1]
```diff
@@ -93,6 +98,33 @@
     "deep": ["exprstmt"],
 }
 
+ALPHA_FEATURES = {
```
Comment (mschwager) [inline scripts/generate_test_matrix.py:101]: I don't think this is set in stone yet. I would hold off here until we've finalized the language maturity doc :+1:

Comment (colleend) [reply scripts/generate_test_matrix.py:101]: ah yes, good catch

**Comment by colleend** [comment]
per Matt's suggestion, I'm going to close this PR so I can separate it into two:

1. For adding in tests and exceptions
2. For adding in the table-changing code

**Pull request closed without merging**